test/e2e/e2eTests.js

```diff
@@ -96,7 +96,7 @@ casper.test.begin('Back to login page to login', 5, function suite(test) {
         // 'true/false' is whether to submit form
         this.fill('form.login-form', {
             'username': "gsadmin",
-            'password': "solinea"
+            'password': "admin"
```

Comment:
For future consideration:  can references to usernames/passwords be variables that are loaded from environment or command-line parameters?  Minimally, pulled up to the top of the file to allow for easier editing?